app/javascript/packs/session-timeout-ping.ts

```diff
@@ -65,25 +36,18 @@ function success(data: SessionStatus) {
     });
   }
 
-  if (timeRemaining > 0 && timeRemaining < frequency) {
```

Comment:
May want to reincorporate safeguards from #8117. We don't have the same "fallback" as mentioned there in this implementation, but we should still check that `timeRemaining` is an expected (`>0`) value.

Reply:
Improved in 9484744c0, uses `timeRemaining` only if in the valid range, otherwise `frequency`.